src/bind.js

```diff
@@ -18,8 +18,17 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...arg) {
+  const arr = [...arg];
+  return (...numbers) => {
```

Comment:
good, but you can write simply
```suggestion
  return (...numbers) => callback(...arg, ...numbers);
```

Reply:
дякую, але тепер я не розуію як воно працює